R/BoxCox.R

```diff
@@ -1,8 +1,7 @@
 #' Box-Cox Transformation for Non-Negative Data
 #'
-#' `step_BoxCox` creates a *specification* of a recipe
-#'  step that will transform data using a simple Box-Cox
-#'  transformation.
+#' `step_BoxCox()` creates a *specification* of a recipe step that will transform
+#' data using a Box-Cox transformation.
```

Comment:
Googled this and it does seem to be the case, but just confirming: "simple" here is a filler word that doesn't clarify meaning and is not a technical term describing the procedure more exactly?

Reply:
simple is filler in this case! :)